map/src/map/layers/LocalClientTrackLayer.js

```diff
@@ -561,7 +574,8 @@ export default function LocalClientTrackLayer() {
         } else if (ctx.selectedGpxFile?.dragPoint === undefined) {
             ctx.trackState.block = false;
             map.getContainer().style.cursor = 'crosshair';
-            map.on("click", clickMap);
+            map.off('click'); // !!!
```

Comment:
The method is called addClickOnMap and we only add the click, not remove it

Reply:
renamed to setClickOnMap, because it's called more than once anyway